File: engine/model/src/main/scala/tools/packet/ui/FinishedSort.scala
package tools.packet.ui

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class FinishedSort(private var _inventory: Int) extends PacketInput {
  def inventory: Int = _inventory

  override def opcode(): SendOpcode = SendOpcode.GATHER_ITEM_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/ui/FinishedSort.java
package tools.packet.ui;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record FinishedSort(int inventory) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GATHER_ITEM_RESULT;
   }
}